crates/rosty/tests/shutdown_token.rs

```diff
@@ -1,4 +1,4 @@
-mod util;
+pub mod util;
```

Comment:
Why does this need to be pub?

Reply:
So that there are no warnings when compiling. Because Rust treats every test file as a different compilation unit, you can get warnings if the functions are not being used in *each* test file.